README.md

```diff
@@ -1,6 +1,6 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+In order for this program to run, should you fork it, two .env files should be created locally - .env.nc_news and .env.nc_news_test
```

Comment:
Remember the names for the .env files is based on what your ENV variable is set as - test or development - rather than the actual name of the databases :)

Reply:
yeah edited the readme before actually doing what i was about to do ahaha
Ill edit again